tools/scripts/generate_binary_build_matrix.py

```diff
@@ -361,6 +369,11 @@ def generate_wheels_matrix(
         # Define default compute architectures
         arches = ["cpu"]
 
+        if os == "linux-aarch64":
```

Comment:
So should with_cuda, with_rocm, etc. be set to false automatically if os = `linux-aarch64`?

Reply:
Hm I don't think its really needed, it will complicate logic further, since we don't actually set with_cuda or with_rocm anywhere in this script, we treat these as our input only variables.

Reply:
They're enabled by default though right? Feel free to structure this however you see fit, as long as the correct matrix is generated (maybe we should have a unit-test for this in a follow-up).